glide.yaml

```diff
@@ -12,3 +12,6 @@ import:
 - package: github.com/pkg/errors
   version: ~0.7.1
 - package: github.com/spf13/cobra
+  version: 9c28e4bbd74e5c3ed7aacbc552b2cab7cfdfe744
+- package: github.com/spf13/pflag
+  version: 7b17cc4658ef5ca157b986ea5c0b43af7938532b
```

Comment:
I add it in comment:
![image](https://cloud.githubusercontent.com/assets/1716915/19954511/cd05c458-a1b3-11e6-8ef5-3c96bdfa64d1.png)
Do you mean add this note in git commit message?